.github/workflows/release.yaml

```diff
@@ -0,0 +1,135 @@
+# Copyright the Hyperledger Blockchain Explorer contributors. All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+name: Build docker image
+
+on:
+  workflow_dispatch: # workflow_dispatch must be enabled in main branch to support release action on older release branches
+  create:
+    tags:
+      - v*
+  push:
+    branches:
+      - main
```

Comment:
Hi @adityajoshi12 , As per yesterdays (23Aug2023) discussion on this PR, the image build should be triggered only whenever a new release happens(build-and-push-docker-release).
Please look into this and make changes accordingly. Thanks.

Reply:
There are two flow in this CI, one is(L62) when anything pushed to the main branch container images will be created with the latest tag, while the other flow(L84) will be invokes on the creation of tags and new container images will be created based on the tag

Reply:
yes, as per the first workflow the image gets created whenever the code is pushed to main branch. As per discussion conclusion yesterday, the build image should happen whenever we create a new release.

Reply:
okay, I understood that, but does that mean we will not have images with latest tag.

Reply:
I think we can run the two workflows. With the first workflow, we would have the images with the latest tag and with the second work flow will have the images with the release tag.  However the first - workflow shouldn't be triggered on every push to main. @adityajoshi12 @arsulegai @vineeth2328  Please suggest.

Reply:
We can have a seperate branch `dev` and all the new commit/PR can be merged to that branch (dev) and after sometime we can merge the dev branch to the main branch. And image with tag latest can be build with main branch

Reply:
I agree @ArchanaArige , for the time being, let us have a human triggered workflow for the `latest` tag image creation. Re: Branching strategy, Manoj (mentee) is experimenting something similar and listing down advantages/disadvantages in terms of maintainability.

Reply:
Will discuss about this one in the next call and make a decision. There are some other dependency PRs, once they are merged, then we can go ahead and merge this one too.